code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -976,4 +982,36 @@
 	loc = H
 	H.ContractDisease(P)
 	parrot_interest = null
-	H.visible_message("<span class='danger'>[src] dive bombs into [H]'s chest and vanishes!</span>", "<span class='userdanger'>[src] dive bombs into your chest, vanishing! This can't be good!</span>")
\ No newline at end of file
+	H.visible_message("<span class='danger'>[src] dive bombs into [H]'s chest and vanishes!</span>", "<span class='userdanger'>[src] dive bombs into your chest, vanishing! This can't be good!</span>")
+
+//Clockwork Hawk
+
+/mob/living/simple_animal/parrot/clockhawk
+	name = "Clockwork Hawk"
+	desc = "The Justicar has always loved aviaries, for they needed no machinery to engage in such a marvelous feat. For this, he gave a select few the ability to see beyond the sky."
+	icon = 'icons/mob/clockwork_mobs.dmi'
+	icon_state = "clockhawk_fly"
+	icon_living = "clockhawk_fly"
+	icon_dead = ""
+	icon_sit = "clockhawk_sit"
+	density = 0
+	health = 80
+	maxHealth = 80
+	pass_flags = PASSTABLE | PASSMOB

```

Comment:
density = 0, health = 80, maxHealth = 80, and pass_flags = PASSTABLE | PASSMOB are all inherited from its parent type.